blog/admin.py

```diff
@@ -1,4 +1,5 @@
 """Configuration for admin Blog app."""
-# from django.contrib import admin
+from django.contrib import admin
+from .models import Post
```

Comment:
Plz separate project imports from the framework imports with one empty line.

Reply:
ok, thanks